GameData/KerbalismConfig/System/Science/HardDriveConfigs.cfg

```diff
@@ -181,6 +181,12 @@ KERBALISM_HDD_SIZES
 		samples = 10
 	}
 
+	dragon_cst
+	{
+		data = 64000								//64 GB, SDD ~ Guess due to date of capsules & tech
+		samples = 3
```

Comment:
A bit short on samples? These capsules can return a lot of mass, definitely more than 3 samples worth.